app/models/rubygem.rb

```diff
@@ -86,10 +86,14 @@ def self.current_rubygems_release
   end
 
   def self.news(days)
-    includes(:latest_version, :gem_download)
-      .with_versions
+    joins(:latest_version)
       .where("versions.created_at BETWEEN ? AND ?", days.ago.in_time_zone, Time.zone.now)
-      .order("versions.created_at DESC")
+      .group(:id)
+      .order("MAX(versions.created_at) DESC")
+  end
```

Comment:
I took `with_versions` out because from what I can tell, `versions` records that get set to `latest` are necessarily indexed.